### ISSUE TITLE
[Component] [select, option] Voice over and windows narrator does not read options in el-select

### ISSUE THREAD

**Issue by RomanRadaykinFrontend** [element-plus/element-plus#14437]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.3.4`
- Element Plus Version: `2.3.14`
- Browser / OS: `MacOS 13.5.2 (Chrome 117.0.5938.92) and Parallels with Windows 10.0.19045 (Chrome 117.0.5938.92)`
- Build Tool: `Webpack`

### Related Component

- `el-select`
- `el-option`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYsIHZlcnNpb24gYXMgdnVlVmVyc2lvbiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgZXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgRWxlbWVudFBsdXMgfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcblxuY29uc3QgbXNnID0gcmVmKCdIZWxsbyBXb3JsZCEnKVxuY29uc3Qgb3B0aW9ucyA9IFtcInRlc3QxXCIsIFwidGVzdDJcIiwgXCJ0ZXN0M1wiXVxuY29uc3QgZm9ybVZhbCA9IHJlZigge1xuICAgIHNlbGVjdDogXCJcIixcbn0pXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtZm9ybSBsYWJlbC1wb3NpdGlvbj1cInRvcFwiIDptb2RlbD1cImZvcm1WYWxcIj5cbiAgICAgIDxlbC1mb3JtLWl0ZW0gdGFiaW5kZXg9XCIwXCIgbGFiZWw9XCJzZWxlY3RcIj5cbiAgICAgICAgPGVsLXNlbGVjdCB0YWJpbmRleD1cIjBcIiBwbGFjZWhvbGRlcj1cInNlbGVjdFwiIHYtbW9kZWw9XCJmb3JtVmFsLnNlbGVjdFwiPlxuICAgICAgICAgIDxlbC1vcHRpb25cbiAgICAgICAgICAgIHRhYmluZGV4PVwiMFwiXG4gICAgICAgICAgICA6YXJpYS1sYWJlbD1cImlcIlxuICAgICAgICAgICAgdi1mb3I9XCJpIGluIG9wdGlvbnNcIlxuICAgICAgICAgICAgOmtleT1cImlcIlxuICAgICAgICAgICAgOnZhbHVlPVwiaVwiXG4gICAgICAgICAgICA6bGFiZWw9XCJpXCJcbiAgICAgICAgICAvPlxuICAgICAgICA8L2VsLXNlbGVjdD5cbiAgICAgIDwvZWwtZm9ybS1pdGVtPlxuICAgIDwvZWwtZm9ybT5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcImFsbG93SnNcIjogdHJ1ZSxcbiAgICBcImNoZWNrSnNcIjogdHJ1ZSxcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWUsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl1cbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJfbyI6e319)

### Steps to reproduce

Enable voice over or windows narrator. Via TAB walk to el-select. Press enter. Flip through the el-option.

### What is Expected?

Expected behavior - the voice will speak the el-option text.

### What is actually happening?

The text is not spoken

### Additional comments

_(empty)_

**Comment by tolking** [comment]
There is no need to configure the `tabindex` or `aria-label` attribute when using it.

**Comment by RomanRadaykinFrontend** [comment]
Okay, but it don't work with tabindex and without it
